packages/firebase/index.d.ts

```diff
@@ -6941,106 +6941,236 @@ declare namespace firebase.messaging {
+     * The returned promise rejects if the user does not allow the app to show notifications.
+     *
+     * @param options.vapidKey The public server key provided to push services. It is used to
+     * authenticate the push subscribers to receive push messages only from sending servers that
+     * hold the corresponding private key. If it is not provided, a default VAPID key is used. Note
+     * that some push services (Chrome Push Service) require a non-default VAPID key. Therefore, it
+     * is recommended to generate and import a VAPID key for your project with
+     * {@link https://firebase.google.com/docs/cloud-messaging/js/client#configure_web_credentials_with_fcm Configure Web Credentials with FCM}.
+     * See
+     * {@link https://developers.google.com/web/fundamentals/push-notifications/web-push-protocol The Web Push Protocol}
+     * for details on web push services.}
+     *
+     * @param options.serviceWorkerRegistration The service worker registration for receiving push
+     * messaging. If the registration is not provided explicitly, you need to have a
+     * `firebase-messaging-sw.js` at your root location. See
+     * {@link https://firebase.google.com/docs/cloud-messaging/js/client#retrieve-the-current-registration-token Retrieve the current registration token}
+     * for more details.
+     *
+     * @return The promise resolves with an FCM registration token.
      *
-     * @return The promise resolves with the FCM token string.
      */
-    getToken(): Promise<string>;
+    getToken(options?: {
+      vapidKey?: string;
+      serviceWorkerRegistration?: ServiceWorkerRegistration;
+    }): Promise<string>;
+
     /**
-     * When a push message is received and the user is currently on a page
-     * for your origin, the message is passed to the page and an `onMessage()`
-     * event is dispatched with the payload of the push message.
-     *
-     * NOTE: These events are dispatched when you have called
-     * `setBackgroundMessageHandler()` in your service worker.
+     * When a push message is received and the user is currently on a page for your origin, the
+     * message is passed to the page and an `onMessage()` event is dispatched with the payload of
+     * the push message.
      *
      * @param
      *     nextOrObserver This function, or observer object with `next` defined,
      *     is called when a message is received and the user is currently viewing your page.
-     * @return To stop listening for messages
-     *    execute this returned function.
+     * @return To stop listening for messages execute this returned function.
      */
     onMessage(
       nextOrObserver: firebase.NextFn<any> | firebase.Observer<any>,
       error?: firebase.ErrorFn,
       completed?: firebase.CompleteFn
     ): firebase.Unsubscribe;
+
+    /**
+     * Called when a message is received while the app is in the background. An app is to be in the
+     * background if no active window is displayed.
```

Comment:
Missing one word here "considered"

"An app is considered to be in the
     * background"